python/sglang/lang/ir.py

```diff
@@ -63,6 +65,7 @@ def to_openai_kwargs(self):
             warnings.warn("Regular expression is not supported in the OpenAI backend.")
         return {
             "max_tokens": self.max_new_tokens,
+            "max_completion_tokens": self.max_completion_tokens,
```

Comment:
Why adding `max_completion_tokens` to the frontend argument, rather than have
```max_completion_tokens": self.max_new_tokens```?
So the SGL language still has only one name, but the function `to_openai_kwargs` takes care of the key mapping.

Reply:
We could do that. I am not sure which one is more user-friendly. e.g. user already knows `max_completion_tokens` is for `o1`. We can probably support both.